### PR TITLE
cloud66/cyclops#864 container friendly name

### PR DESCRIPTION
**Pull request by babakgh**
container command help changed

### CODE REVIEW

File: _posts/toolbelt/containers.md [hunk 1]
```diff
@@ -78,9 +84,9 @@ Stops a particular container on the given stack based on container Id.
         <td>Name of the stack</td>
     </tr>
     <tr>
-        <td><i>container Id</i></td>
+        <td><i>container</i></td>
         <td>&mdash;</td>
-        <td>The container Id</td>
+        <td>The container ID or container name (short version supported)</td>
```
Comment (lvangool) [inline _posts/toolbelt/containers.md]: We should be consistent with "ID" or "Id" - please update (check whether we use "ID" or "Id" on other pages pls)

File: _posts/toolbelt/containers.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ lead: Manage your containers from the toolbelt
 <h2 id="container-list">Listing containers</h2>
 <h3>Usage</h3>
 <pre class="prettyprint">
-$ cx containers list [-s &lt;stack&gt;] [--server &lt;slave server name&gt;|&lt;slave server ip&gt;]
+$ cx containers list [-s &lt;stack&gt;] [--server &lt;slave server name&gt;|&lt;slave server ip&gt;] [--trunc <false|true>]
```
Comment (lvangool) [inline _posts/toolbelt/containers.md]: Default for trunc is "true" now right?

Comment (babakgh) [inline _posts/toolbelt/containers.md]: yes , also It was not in help , I added the option with default true

File: _posts/toolbelt/containers.md (hunk 1 above)
Comment (babakgh) [inline _posts/toolbelt/containers.md]: solved

**Pull request merged**